content/docs/guides/go-application.md

```diff
@@ -82,9 +81,11 @@ func recordMetrics() {
         }()
 }
 
-func main() {
+func init() {
```

Comment:
Put this right beside where the metric is defined, as that makes it more likely to get updated as metrics are added.

There's also promauto which does this automatically.

Reply:
I'm a little on the fence re: `promauto`. I think it's clearly a better way to do things in most cases but may introduce more complexity than we'd want in a guide. I'll defer to you on that question.

Reply:
I think it'd be an easier way for users generally. Manual registration is for more complex use cases.